### DISCUSSION TITLE
Get camera/phone position

### DISCUSSION THREAD

**Discussion by velickolb** [google/model-viewer#2926] [Q&A]
Hello

Is it possible to instantiate model relative to the camera/phone? For example I want to instantiate the model at floor level directly bellow the phone. 

Thanks!

**Comment by elalish** [comment]
Only in WebXR mode. The  initial placement is based on the last camera orientation from the 3D  view. You should be able to change `camera-target` and perhaps set the radius to zero in `camera-orbit` to achieve  what you're after. Give it a try!

**Reply by velickolb** [reply]
Thanks, will try it!
